src/node_http2.h

```diff
@@ -1075,7 +1082,8 @@ class Http2Settings : public AsyncWrap {
       size_t count,
       const nghttp2_settings_entry* entries);
 
-  Http2Session* session_;
+  BaseObjectWeakPtr<Http2Session> session_;
+  v8::Persistent<v8::Function> callback_;
```

Comment:
These callbacks also need to be tracked through `MemoryInfo()` implementations now, because V8 cannot see the JS property on the object anymore.